custom_model_runner/datarobot_drum/drum/language_predictors/python_predictor/python_predictor.py

```diff
@@ -72,7 +72,7 @@ def _dr_api_url(endpoint):
             endpoint = f"{endpoint}/api/v2"
         return endpoint
 
-    def _supports_chat(self):
+    def supports_chat(self):
```

Comment:
is this tested anywhere?  a quick search yielded no tests for this.  

- if it's tested, please provide a link
- if not, please test this check of the custom hooks

Reply:
Test `supports_chat` or `has_custom_hook` ?

Reply:
Added a test for `supports_chat`